packages/components/src/components/tag-dismissible/tag-dismissible-styles.ts

```diff
@@ -4,17 +4,18 @@ import {
   getScreenReaderOnlyJssStyle,
   getThemedColors,
   getTransition,
-  pxToRemWithUnit,
 } from '../../styles';
 import { getCss } from '../../utils';
 import type { TagDismissibleColor } from './tag-dismissible-utils';
-import { textSmallStyle } from '@porsche-design-system/utilities-v2';
+import { borderRadiusSmall, textSmallStyle } from '@porsche-design-system/utilities-v2';
 import { getTagFocusJssStyle, getThemedBackgroundColor, slottedTextJssStyle } from '../tag/tag-styles';
 import { hoverMediaQuery } from '../../styles/hover-media-query';
+import type { Theme } from '../../utils/theme';
+import { hostHiddenStyles } from '../../styles/host-hidden-styles';
 
-export const getComponentCss = (color: TagDismissibleColor, hasLabel: boolean): string => {
-  const themedColors = getThemedColors('light');
-  const { primaryColor, hoverColor, contrastMediumColor } = themedColors;
+export const getComponentCss = (color: TagDismissibleColor, theme: Theme, hasLabel: boolean): string => {
+  const themedColors = getThemedColors(theme);
+  const { primaryColor, hoverColor, contrastHighColor, focusColor } = themedColors;
   const backgroundColor = getThemedBackgroundColor(color, themedColors);
 
   return getCss({
```

Comment:
All of the host styles should be !important now, since we offer the possibility to hide element with hidden attribute